analysis/include/BaseSelectorSVT.h

```diff
@@ -0,0 +1,59 @@
+#ifndef BASESELECTOR_H
```

Comment:
Why is this whole new class needed? This looks like a copy of BaseSelector.h, but without the nice comments for the Doxygen documentation. Is it possible to achieve this class by inheriting from BaseSelector? It is unclear to me what this has that BaseSelector does not.

Reply:
Fixed. It again was an early artefact of getting things working. I had deleted the cpp file, and forgotten to remove the corresponding h file. Have done now.